document-portal/file-transfer.c

```diff
@@ -174,7 +174,7 @@ file_transfer_start (XdpAppInfo *app_info,
     g_free (transfer->key);
     key = g_random_int ();
     key = (key << 32) | g_random_int ();
-    transfer->key = g_strdup_printf ("%lu", key);
+    transfer->key = g_strdup_printf ("%" G_GUINT64_FORMAT, key);
```

Comment:
This is really a somewhat separate issue (it's `guint64`, not `fuse_ino_t`) and is certainly correct.

Reply:
Unlike the FUSE inode numbers, this was never portable, so please separate it into its own commit, perhaps something like "file-transfer: Use correct formatting specifier for guint64".